addons/project_todo/static/tests/todo_form_view.test.js

```diff
@@ -23,7 +23,7 @@ beforeEach(() => {
                 <field name="name" nolabel="1"/>
                 <field name="state" widget="todo_done_checkmark" nolabel="1"/>
             </tree>`,
-        form: `
+        "form,1": `
```

Comment:
```suggestion
        "form, 1": `
```

Reply:
By applying space, it cannot find the view because it considers the space also. So, it does not need to change.